lib/ricohapi/mstorage/client.rb

```diff
@@ -95,7 +95,18 @@ def add_meta(media_id, user_meta)
 
       # DELETE /media/{id}/meta/user, DELETE /media/{id}/meta/user/{key}
       def remove_meta(media_id, key)
-        # TODO: do something
+        case key
+        when 'user'
+          handle_response(:as_raw) do
+            token.delete endpoint_for("media/#{media_id}/meta/user")
+          end
+        when USER_META_REGEX
+          handle_response(:as_raw) do
+            token.delete endpoint_for("media/#{media_id}/meta/user/#{$1}")
+          end
+        else
+          raise Error.new("invalid key: #{key}")
```

Comment:
I saw your test result screenshot, I think following code is better: 
```ruby
raise Error.new("invalid key: #{key.inspect}")
```
If it is the  above code, we can know the key is either nil or "".